tls/s2n_security_policies.c

```diff
@@ -704,17 +599,10 @@ const struct s2n_security_policy security_policy_pq_tls_1_0_2021_05_24 = {
     },
 };
 
-const struct s2n_security_policy security_policy_pq_tls_1_0_2021_05_25 = {
-    .minimum_protocol_version = S2N_TLS10,
-    .cipher_preferences = &cipher_preferences_pq_tls_1_0_2021_05_25,
-    .kem_preferences = &kem_preferences_pq_tls_1_0_2021_05,
-    .signature_preferences = &s2n_signature_preferences_20140601,
-    .ecc_preferences = &s2n_ecc_preferences_20200310,
-};
-
+/* Used by AWS CRT SDK */
 const struct s2n_security_policy security_policy_pq_tls_1_0_2021_05_26 = {
     .minimum_protocol_version = S2N_TLS10,
-    .cipher_preferences = &cipher_preferences_pq_tls_1_0_2021_05_26,
+    .cipher_preferences = &cipher_preferences_20210825_gcm,
```

Comment:
Why does this not change the security policy? Why is this safe? (Same for the other policies)